CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # spring-service
 
+## 3.1.2
+
+- Increase startupProbe time until a failure occurs to 2 minutes.
```

Comment:
Smallish side question: I would guess the total time until a pod is killed is now 2:30 (24 startup probes à 5 + 3 liveness probes à 10 s). Or does it get immediately killed, if the startup probe never is successful?

Reply:
It will be killed just after the startup probes time out. Because the readiness and liveness probes will never be armed. (They are only armed after successful startup probe)

> The kubelet uses startup probes to know when a container application has started. If such a probe is configured, it disables liveness and readiness checks until it succeeds, making sure those probes don't interfere with the application startup. This can be used to adopt liveness checks on slow starting containers, avoiding them getting killed by the kubelet before they are up and running.